src/whenever/__init__.py

```diff
@@ -2306,10 +2306,36 @@ def _unpkl_naive(*args) -> NaiveDateTime:
 class Ambiguous(Exception):
     """A datetime is unexpectedly ambiguous"""
 
+    @staticmethod
+    def for_timezone(d: _datetime, tz: _tzinfo) -> Ambiguous:
+        return Ambiguous(
+            f"{d.replace(tzinfo=None).isoformat()} is ambiguous "
```

Comment:
while `isoformat` is nice, we can also just use the default string representation. It omits the `T` which is a bit more readable:

```f"{d.replace(tzinfo=None)} is ambiguous"```